### DISCUSSION TITLE
Visual Studio Code extension -- Subscription Required

### DISCUSSION THREAD

**Discussion by jayliang42** [community/community#43819] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I have student account but the vscode keep showing subscription required. I tried restart vscode, reinstall copilot.
<img width="708" alt="image" src="https://user-images.githubusercontent.com/87124279/211498537-5c0f7da8-198c-4090-b307-63fc4255f05a.png">

**Comment by jongwooo** [comment]
@zhisongliang This is just a badge for the README. Please check [here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot&ssr=false#overview).

**Reply by jayliang42** [reply]
Thank you for the quick response!
